src/sentry/tasks/update_user_reports.py

```diff
@@ -63,7 +65,12 @@ def update_user_reports(**kwargs: Any) -> None:
         for event in events:
             report = report_by_event.get(event.event_id)
             if report:
-                reports_with_event += 1
+                if features.has(
+                    "organizations:user-feedback-ingest", project.organization, actor=None
+                ):
+                    shim_to_feedback(
+                        report, event, project, FeedbackCreationSource.USER_REPORT_ENVELOPE
+                    )
                 report.update(group_id=event.group_id, environment_id=event.get_environment().id)
```

Comment:
Should we update these fields before creating the feedback? I.e. move it above line 68

Reply:
i don't think it really matters, what do you think?